### PR TITLE
Fix "pushObjects is not a function" error

### LINKED ISSUE #1099: Latest updates in dashboard don't show up
The HTTP request succeeds but the items are never displayed (the "My crates" and "Following" sections work as expected, but not the "Latest updates").

The console logs this error: "e.get(...).pushObjects is not a function"

### PR DESCRIPTION
**Pull request by locks**
This is a bit of a quick patch to address the problem, we should dig some more into why it's happening.

Fixes #1099

[edit] I only checked `config/environment.js` for something disabling prototype extensions, and forgot to check `package.json`, which lists [ember-disable-prototype-extensions](https://github.com/rust-lang/crates.io/blob/master/package.json#L49).

### CODE REVIEW

**Comment by Turbo87** [comment]
this seems necessary only for `myFeed` due to https://github.com/rust-lang/crates.io/blob/dc195a352cd7e336233c0694cbd6729f2b2de053/app/controllers/dashboard.js#L52

**Comment by locks** [comment]
@Turbo87 my thought was let's keep it consistent. We could also do `this.get('myFeed').push(...versions)` if `versions` is a native array.

**Comment by Turbo87** [comment]
we could, but then the array observers won't get notified for new items in the array

**Comment by carols10cents** [comment]
Sorry this has taken me so long to look at-- I'm still getting `_this.get(...).pushObjects is not a function` on my dashboard page :-/

I'm going to see if I can get a test to fail for this...

**Comment by Turbo87** [comment]
> I'm still getting `_this.get(...).pushObjects` is not a function on my dashboard page

even after applying this patch?

**Comment by carols10cents** [comment]
yep, on this branch

**Comment by carols10cents** [comment]
does this branch fix it for you?

**Comment by carols10cents** [comment]
Welp i just tried to make a test with a logged in user for the past few hours and i fail. Pushed my attempt to this branch, feel free to toss it if it's crap.

**Comment by Turbo87** [comment]
@carols10cents @locks I finally got around to looking into this and after a while I managed to figure out how to setup everything so that auth works locally too.

tl;dr we missed `controller.set('myFeed', []);` in the `dashboard` route file

I've fixed the instruction and pushed the fix to this PR. I've also moved your test to a separate `me-test` branch to make CI pass again here, but we should definitely continue to explore how to add tests for these pages.

**Comment by locks** [comment]
@carols10cents can you 👍 or merge?

**Comment by carols10cents** [comment]
Yay it works!!! Thank you!!!

bors: r+
